.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   - "6.0"
 after_success:
-  - npm run coverage
+  - npm run codeclimate-test-reporter
```

Comment:
why not reuse the coverage script since that has a `scripts/coverage.sh` equivalent here: https://github.com/denali-js/denali/blob/master/scripts/coverage.sh